### PR TITLE
Kz/volunteerpics

### PR DESCRIPTION
**Pull request by kimzim**
I think I fixed the issue with the last pull request. Please let me know if you find any other issues

### CODE REVIEW

File: about_us.html [hunk 1]
```diff
@@ -1,13 +1,16 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 
 <head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (JessicaGreben) [inline about_us.html:6]: nice additions :)

**Review by JessicaGreben — APPROVED** [review]
The pics look great, nice changes. The only thing I would change is that you can remove the files `images/.DS_Store` and `images/P5/.DS_Store`.  I added a gitignore file so those shouldn't be a problem in the future, but for your PR, you should remove them

**Comment by kimzim** [comment]
I can't locate the files to delete them. Do you know how I might find them? I have looked in the images file

**Comment by JessicaGreben** [comment]
@kimzim from the `images` directory if you type `ls -a` they should show up.  The other option is to checkout master; pull master, then go back onto your branch and merge master into it. That should add my change which should ignore the DS_Store files

**Comment by kimzim** [comment]
When I go into the images directory it does not show up

**Pull request merged**